README.md

```diff
@@ -21,55 +21,72 @@
 
-> The CachyOS repositories provide prebuilt kernels in three different march versions: `x86-64`, `x86-64-v3`,`x86-64-v4` and `znver4` . In addition, the repositories also offer LTO-enabled kernels.
-> The default `linux-cachyos` kernel is compiled with Clang and ThinLTO. For this kernel there is a `linux-cachyos-gcc` variant available.
+> [!NOTE]
+> The CachyOS repositories provide prebuilt kernels in three different
+> march versions: `x86-64`, `x86-64-v3`,`x86-64-v4` and `znver4`. The default
+> `linux-cachyos` kernel is compiled with Clang and Thin LTO. For this kernel
+> there is a `linux-cachyos-gcc` variant available.
 
 ## Features
 ### :hammer_and_wrench: Advanced building & compiling
+
 - Very customizable PKGBUILD with many features and improvements.
-- `GCC/CLANG` Optimization with automatically found CPU architecture or also selectable CPU architecture.
-- Choose between `LLVM/LTO & Thin-LTO` or `GCC`.
+- Auto detection of CPU architecture and setting the appropriate compiler flags.
+- Support for building with GCC or Clang.
 - Choose between 300Hz, 500Hz, 600 Hz ,750Hz and 1000Hz. Defaults to 1000Hz
 - Kernel Control Flow Integrity (kCFI) selectable when using `LLVM`
+- AutoFDO support
 
 ### :abacus: CPU enhancements
-- 3 Different scheduler are supported,`SCHED-EXT`,`BORE`,`EEVDF` and `BMQ` scheduler
-- AMD P-State Preferred Core / amd-pstate enhancements and fixes from -next.
+
+- 3 different scheduler are supported: `BORE`, `EEVDF` and `BMQ` scheduler
+- AMD P-State Preferred Core / amd-pstate enhancements and fixes from `linux-next`.
+- Back-ported scheduler patches from `linux-next`.
 - AMD 3D V-Cache optimizer driver
-- AMD Hardware Feedback Interface
-- Intel Pstate Hybrid CPU Driver
-- SCHED-EXT Schedulers prebuilt in the repository # https://lwn.net/Articles/922405/ - `linux-cachyos` and `linux-cachyos-sched-ext`
+- Support for RT kernel builds with BORE
 - Cachy Sauce `CONFIG_CACHY`, enables various tweaks for the scheduler and other settings
-- Scheduler patches from `linux-next` in `linux-cachyos-bore` and `linux-cachyos-eevdf`
+
 ### :bookmark_tabs: Filesystem & memory
-- ZFS Filesystem support and prebuilt in the repository
-- NVIDIA Module support including patches - Build the nvidia module together with the kernel
+
+- ZFS Filesystem support and pre-built modules for our kernels in the repository
+- Support for building NVIDIA modules with some patches. Ready modules for
+binary kernel builds can be found in the repository.
 - Latest & improved ZSTD 1.5.6 patch-set
-- UserKSM daemon from pf
-- THP Shrinker - reduces memory usage for transparent hugepages
-- Memory Management Tweaks (LRU, Compaction, Watermark)
+- KSM patch for proper working of uksmd daemon from pf
+- Memory management tweaks from zen-kernel (MGLRU, Compaction, Watermark)
 
 ### &#128423; Network
-- BBRv3 tcp_congestion_control
+
+- Using BBRv3 as the default TCP congestion control algorithm
```

Comment:
I think the plan right now is to revert back to Arch default (cubic) while providing BBRv3 as an option.